src/styles/app.css

```diff
@@ -0,0 +1,15 @@
+.app {
+  display: flex;
+  flex-direction: column;
+}
+
+body {
+  background-color: rgba(0, 0, 0, 0.902)
+}
+
+.nasa-logo {
+  margin: auto;
```

Comment:
Defining margin like this is the same as
```
margin-top: auto;
margin-right: auto;
margin-bottom: auto;
margin-left: auto;
```
But `margin: auto` only works for the horizontal axis so youre declaring things you dont need here. If you want to keep it in shorthand form you can just change this to:
```
margin: 0 auto;
```